pkg/controller/operators/operatorcondition_controller.go

```diff
@@ -98,8 +98,7 @@ func (r *OperatorConditionReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	operatorCondition := &operatorsv2.OperatorCondition{}
 	err := r.Client.Get(context.TODO(), req.NamespacedName, operatorCondition)
 	if err != nil {
-		log.V(1).Error(err, "Unable to find operatorcondition")
-		return ctrl.Result{}, err
```

Comment:
Just for further context, this is the log spamming I was seeing locally:

```
$ cat olm.log | grep "Unable to find operatorcondition" | wc -l
699
```

That olm operator container log had only been running for ~10m and the logging level was at the default info level.

Reply:
Also in this case, it seems sufficient to ignore an error when apierrors.IsNotFound(err) evaluates to true if the operatorconditiongenerator controller is responsible for generating that resource.